src/plugins/email.py

```diff
@@ -103,15 +108,31 @@ def validateConfig(self):
         self.sender = self.cfg["smtp_sender"]
         self.user = self.cfg["smtp_user"]
         self.password = self.cfg["smtp_pass"]
+        if "smtp_nologin" not in self.cfg:
+            self.cfg["smtp_nologin"] = False
+
+        self.nologin = self.cfg["smtp_nologin"]
+
+        if "smtp_sender_name" in self.cfg["smtp_sender_name"]:
```

Comment:
Is here the "not" missing? You are overriding when it is present?

Reply:
Not sure I understand, if the key exists I use it.

Reply:
Oh crap I see it, let me check

Reply:
2 errors, not needed, and sit should not be the key, sicne it would throw an error if not set, check if the key is not in self.cfg, if not set it to None